docusaurus.config.js

```diff
@@ -53,7 +53,7 @@ module.exports = {
                         },
                         {
                             label: "Advanced",
-                            to: "docs/advanced/writing-declarations",
+                            to: "docs/advanced/compiler-annotations",
```

Comment:
this does not look right

Reply:
compiler annotations is the new first element in the list, since writing-declarations dropped down, so I think it is correct